test-common/fonts/MutatorSans.fontra/glyphs/E^1.json

```diff
@@ -8,6 +8,9 @@
 "italic": 0,
 "weight": 150,
 "width": 0
+},
```

Comment:
Together with the redundant tests in tests_backends_fontra.py, this change can be reverted, too.

Reply:
done.